src/NewTools-Playground-Tests/StPlaygroundPagePresenterTest.class.st

```diff
@@ -59,6 +59,17 @@ StPlaygroundPagePresenterTest >> testAddToolbarCommandsTo [
 	self assert: (commands includes: class)
 ]
 
+{ #category : 'tests' }
+StPlaygroundPagePresenterTest >> testContents [
+	|string|
+	string := 'coucou'.
+	self openInstance.
+	
+	self assert: presenter text text equals:''.
```

Comment:
`presenter text text` would not make demeter happy 

(Demeter's law state that we should not show the internal implementation)